### PR TITLE
[UPDATE] System.Reactive to 4.1.5 WIP

### PR DESCRIPTION
**Pull request by lxatstariongroup**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/COMET-SDK-Community-Edition/pulls) open
- [x] I have verified that I am following the COMET-SDK [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/COMET-SDK-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description
[UPDATE] System.Reactive to 4.1.5

### CODE REVIEW

File: CDP4Dal/CDP4Dal.csproj [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <PackageTags>CDP COMET ECSS-E-TM-10-25</PackageTags>
     <PackageLicenseExpression>LGPL-3.0-only</PackageLicenseExpression>
     <PackageReleaseNotes>
-      [Update] to CDP4Common 8.1.0
+      [Update] .Net Framework to System.Reactive 4.1.5
```
Comment (samatrhea) [inline CDP4Dal/CDP4Dal.csproj:22]: please rephrase to 
[Update] System.Reactive to version 4.1.5
[Drop] support for net45;net451;net452

File: CDP4Common/CDP4Common.csproj [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net45;net451;net452;net46;net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net46;net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
```
Comment (samatrhea) [inline CDP4Common/CDP4Common.csproj:4]: please add to the PackageReleaseNotes:

[Drop] support for net45;net451;net452

File: CDP4Dal/CDP4Dal.csproj [hunk 3]
```diff
@@ -40,14 +40,12 @@
     <ProjectReference Include="..\CDP4Common\CDP4Common.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net45' Or '$(TargetFramework)' == 'net451' Or '$(TargetFramework)' == 'net452' Or '$(TargetFramework)' == 'net46' Or '$(TargetFramework)' == 'net461' Or '$(TargetFramework)' == 'net462' Or '$(TargetFramework)' == 'net47' Or '$(TargetFramework)' == 'net471' Or '$(TargetFramework)' == 'net472' Or '$(TargetFramework)' == 'net48' ">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net46' Or '$(TargetFramework)' == 'net461' Or '$(TargetFramework)' == 'net462' Or '$(TargetFramework)' == 'net47' Or '$(TargetFramework)' == 'net471' Or '$(TargetFramework)' == 'net472' Or '$(TargetFramework)' == 'net48' ">
     <Reference Include="System.ComponentModel.Composition" />
-    <PackageReference Include="Rx-Core" version="2.2.5" />
```
Comment (samatrhea) [inline CDP4Dal/CDP4Dal.csproj:45]: nice that these can be dropped!

File: CDP4JsonFileDal/CDP4JsonFileDal.csproj [hunk 4]
```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net45;net451;net452;net46;net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net46;net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
```
Comment (samatrhea) [inline CDP4JsonFileDal/CDP4JsonFileDal.csproj:4]: please add to the PackageReleaseNotes:

[Drop] support for net45;net451;net452

File: CDP4JsonSerializer/CDP4JsonSerializer.csproj [hunk 5]
```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net45;net451;net452;net46;net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net46;net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
```
Comment (samatrhea) [inline CDP4JsonSerializer/CDP4JsonSerializer.csproj:4]: please add to the PackageReleaseNotes:

[Drop] support for net45;net451;net452

File: CDP4RequirementsVerification/CDP4RequirementsVerification.csproj [hunk 6]
```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net45;net451;net452;net46;net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net46;net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
```
Comment (samatrhea) [inline CDP4RequirementsVerification/CDP4RequirementsVerification.csproj:4]: please add to the PackageReleaseNotes:

[Drop] support for net45;net451;net452

File: CDP4Rules/CDP4Rules.csproj [hunk 7]
```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net45;net451;net452;net46;net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net46;net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
```
Comment (samatrhea) [inline CDP4Rules/CDP4Rules.csproj:4]: please add to the PackageReleaseNotes:

[Drop] support for net45;net451;net452

File: CDP4ServicesDal/CDP4ServicesDal.csproj [hunk 8]
```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net45;net451;net452;net46;net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net46;net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
```
Comment (samatrhea) [inline CDP4ServicesDal/CDP4ServicesDal.csproj:4]: please add to the PackageReleaseNotes:

[Drop] support for net45;net451;net452

File: CDP4WspDal/CDP4WspDal.csproj [hunk 9]
```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net45;net451;net452;net46;net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net46;net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
```
Comment (samatrhea) [inline CDP4WspDal/CDP4WspDal.csproj:4]: please add to the PackageReleaseNotes:

[Drop] support for net45;net451;net452

**Review by samatrhea — CHANGES_REQUESTED** [review]

**Review by samatrhea — APPROVED** [review]

**Comment by samatstariongroup** [comment]
in the meantime updated to 6.0.1

**Pull request closed without merging**